tests/infra_test.go

```diff
@@ -678,8 +680,9 @@ var _ = Describe("[Serial]Infrastructure", func() {
 				Transport: tr,
 			}
 
-			errors := make(chan error)
+			errorsChan := make(chan error, concurrency)
 			for ix := 0; ix < concurrency; ix++ {
+				defer GinkgoRecover()
```

Comment:
shouldn't the defer exist inside the goroutine? see [example](https://onsi.github.io/ginkgo/#marking-specs-as-failed)

Reply:
totally